packages/termux-x11/termux-x11.c

```diff
@@ -7,7 +7,7 @@
 #include <wayland-client.h>
 
 #ifndef TERMUX_PREFIX
-# define TERMUX_PREFIX "/data/data/com.termux/files/usr"
+# define TERMUX_PREFIX "$PREFIX"
```

Comment:
Use `@TERMUX_PREFIX@` and `sed` instead.

Reply:
Yes i noticed about it.

Reply:
Thinking about it, This should be ignored as the compiler command already provide `-DTERMUX_PREFIX`.

As the result, i will revert the commit.